packages/react-components/react-button/library/src/components/MenuButton/useMenuButtonStyles.styles.ts

```diff
@@ -24,8 +25,22 @@ const useRootExpandedStyles = makeStyles({
   // Appearance variations
   outline: {
     ...shorthands.borderColor(tokens.colorNeutralStroke1Selected),
-    ...shorthands.borderWidth(tokens.strokeWidthThicker),
+    ...shorthands.borderWidth(semanticTokens.strokewidthCtrlOutlineSelected),
```

Comment:
Might be more efficient to create a style and apply it to all three states vs triplicating it.